include/BSMPT/bounce_solution/bounce_solution.h

```diff
@@ -415,9 +523,25 @@ class BounceSolution
   void SetGstar(const double &gstar_in);
 
   /**
-   * @brief GetGstar Get gstar
+   * @brief Generate the spline used to interpolate the gstar SM profile
+   *
+   */
+  void InitializeGstarProfile();
+
+  /**
+   * @brief Initialize two splines for the potential across the tunneling
+   * profile. Used to improve the Hubble rate calculation speed
+   *
+   */
+  void InitializedVSpline();
+
+  /**
+   * @brief Get the Gstar object
+   *
+   * @param T temperature
+   * @return double
    */
-  double GetGstar() const;
+  double GetGstar(const double &T) const;
```

Comment:
Maybe a default value or an overload for old calls?

Reply:
We added an overload for the call with no argument with `CalcGstarPureRad` (effective degrees of freedom in the fully thermalized universe). This way, `GetGstar()` returns the same value as before, and `GetGstar(T)` gives the new temperature-dependent value.